evap/student/templates/student_vote.html

```diff
@@ -101,11 +104,29 @@
             // hide questionnaire for contributor
             $("#" + contributorId).collapse("hide");
             $("a[href=#" + contributorId + "]").addClass("collapsed");
+
+            // setting the 'checked' prop does not trigger a change event, so manually save all data here
+            sisyphus.saveAllData()
         }
         function selectedAnswer(fieldName) {
             // remove error highlighting when an answer was selected
             $(".choice-error[name=" + fieldName + "]").removeClass("choice-error");
         }
         preventAccidentalClosing = true;
+
+        $(function() {
+            sisyphus = $("#student-vote-form").sisyphus({
+                locationBased: true,
+                excludeFields: $("input[type=hidden]"), // exclude the csrf token
+                onRestore: function () {
+                    // If the form contains errors, is submitted, then changes are made and the page is reloaded,
+                    // some labels will have been set active by the server but the underlying checkbox is no longer
+                    // checked because it has been updated by sisyphus
```

Comment:
these comments were not perfectly understandable to me at first. can you please just write a more general `update vote button states` before the two lines of code?